### DISCUSSION TITLE
Photoprism does not find my pictures

### DISCUSSION THREAD

**Discussion by huber75** [photoprism/photoprism#3531] [Q&A]
Good day,

installed PP 3 days ago with success on a Raspberry 4B, 8GB, databank was created and  it took a day to read the directory and display >14,000 pics, Everything fine.

A day later I lost my connection in the net 192.xxx.xxx.xx.:2342 and I installed PP again, could then enter the website again. I added a folder with some pics to the directory on the PI and started the index, but no further album/folder appeared. After many trials I now have no pictures in my photoprism website anymore and the index search results always in one folder where 2 pics are.

I guess that PP looks for the pictures library on the PI and does not find it and I'm not able to give PP the right answer

I'm a really newbie PP concerned and kindly ask for some help, thanks Huber

**Comment by lastzero** [comment]
It could be that you didn't mount an originals folder, the wrong folders are mounted, there's a problem with the database or the filesystem permissions. Also see the Troubleshooting Checklists we provide and note that we can't provide additional help at this time as our team is on vacation:

https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

**Comment by huber75** [comment]
thanks for reply, the checklist does not give me an answer I can live with, I try to get the whole library but always just reach one single folder with 2 pics and that's it

Library path at the PI is /home/pi/Pictures/

**Comment by huber75** [comment]
Good day,

nobody there to assist, please?

In the meantime I'm running a fresh installation bullseye 64-bit sd card on PI4B, 8GB and installed docker, docker-compose and photoprism.
I can enter via IP xxx.xxx.xxx.xx:2432 the program where I try to indicate/import trial files by folder and always get the report "original directory is empty"

**Reply by lastzero** [reply]
Still on vacation, also the community support is provided by the community.

**Comment by huber75** [comment]
@lastzero:

??? Aren't there any user within the community worldwide to participate in a forum, neither english nore german?

**Comment by graciousgrey** [comment]
The path PhotoPrism is reading files from is defined in the docker-compose.yml within the volumes section. If you have installed PhotoPrism as described here:

https://docs.photoprism.app/getting-started/raspberry-pi/microsd-image/

the default path is /opt/photoprism/originals. In the docker-compose.yml it looks like this:

 `volumes:
      - "/opt/photoprism/originals:/photoprism/originals" # Original media folder (DO NOT REMOVE)

If you add photos to /opt/photoprism/originals and PhotoPrism has the right [permissions](https://docs.photoprism.app/getting-started/troubleshooting/docker/#file-permissions) files should appear in the Web UI after indexing.

Can you share how the volumes section looks like in your docker-compose.yml?

**Comment by huber75** [comment]
Hallo Theresa,

danke für die Rückantwort, das "Problem" ist mittlerweile über ein 
anderes Forum gelöst

Ist der Betrag von 24,00 Euro "Essential" wieder aktiv? Hatte 
geschrieben, dass ich nun doch das Essential-Paket nehme

lg, Peter
